designer/server/src/routes/account/auth.test.js

```diff
@@ -10,6 +10,7 @@ jest.mock('~/src/lib/forms.js')
 describe('Authentiation', () => {
   /** @type {import('@hapi/hapi').Server} */
   let server
+  const testingScopes = [auth, authScopesEmpty, authGroupsInvalid]
```

Comment:
If we're only using this for a single `describe.each()` can we put it inside?

```js
  describe.each(
    auth,
    authScopesEmpty,
    authGroupsInvalid
  )('With valid or invalid scopes', (authIn) => {
```

Reply:
I am getting type errors, is it worth adding types in test? or shall i leave it as it is now?

Reply:
Sorry I missed the wrapping `[]` in my suggestion above (fixed)

Reply:
ofcourse :), I was not paying enough attention to fix that myself :)

Reply:
done